CONTRIBUTING.md

```diff
@@ -38,8 +38,7 @@ and answer the questions so we can understand and reproduce the
 problematic behavior.
 
 To show us that the issue you are having is in Gitea itself, please
-write clear, concise instructions so we can reproduce the behavior
-(even if it seems obvious). The more detailed and specific you are,
+write clear, concise instructions so we can reproduce the behavior—even if it seems obvious. The more detailed and specific you are,
```

Comment:
But we just adjust to 80 columns one line.

Reply:
Ah, okay. Yeah, I noticed some portions of the file have manual breaks and some portions don't. I'll go ahead and change that.

Might be worth adding that to the guidelines in the future, too—unless I missed it.

Reply:
@lunny I went ahead and wrapped all of the lines I changed to 80 columns. Should be okay now!